operations/iuf/stages/management_nodes_rollout.md

```diff
@@ -136,3 +137,83 @@ Expected behavior:
 1. Worker nodes `ncn-w001,ncn-w002,ncn-w003` will be upgraded
 1. Worker nodes `ncn-w005,ncn-w006,ncn-w007` will be upgraded
 1. Worker nodes `ncn-w008,ncn-w009,ncn-w010` will be upgraded
+
+## Set NCN boot image for NCN master and NCN storage nodes
+
+These steps should be followed when upgrading NCN storage nodes and `ncn-m001` during [Management-nodes-rollout with CSM upgrade](../workflows/upgrade_all_products.md#651-management-nodes-rollout-with-csm-upgrade)
+when following the [upgrade all products documentation](../workflows/upgrade_all_products.md).
+
+1. (`ncn-mw#`) Set the `IMS_RESULTANT_IMAGE_ID` to be the `final_image_id` found in [Management-nodes-rollout with CSM upgrade](../workflows/upgrade_all_products.md#651-management-nodes-rollout-with-csm-upgrade) in the `upgrade all products documentation`.
+
+    ```bash
+    IMS_RESULTANT_IMAGE_ID=final_image_id
```

Comment:
I fear someone in a hurry could misinterpret this and just copy-paste as is.  Is something like tolerable? 
```
    IMS_RESULTANT_IMAGE_ID=< value of final_image_id >
```